src/shared.ts

```diff
@@ -107,7 +108,12 @@ export const sessionCachedFetch = async <T>(
 	const sessionStorage = await storage.session.get(key);
 	const data = sessionStorage[key] as CachedFetchResponse<T> | undefined;
 	if (data && data.timestamp > Date.now() - cacheTimeMinutes * 60 * 1000) {
+		debug('Cache hit:', key);
 		return data.data;
+	} else if (data) {
+		debug('Cache stale:', key);
+	} else {
+		debug('Cache miss:', key);
```

Comment:
I'm good with keeping this in, just wanted to double check that you intended to keep this

Reply:
yeah, it doesn't log anything if the mode is production